src/microsoft/office/license/user/order/microsoft-office-license-user-order.controller.js

```diff
@@ -0,0 +1,43 @@
+angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicenseUserOrderCtrl", class MicrosoftOfficeLicenseUserOrderCtrl {
+
+    constructor (Alerter, MicrosoftOfficeLicenseService, $rootScope, $scope, OvhHttp) {
+        this.alerter = Alerter;
+        this.licenseService = MicrosoftOfficeLicenseService;
+        this.$rootScope = $rootScope;
+        this.$scope = $scope;
+        this.OvhHttp = OvhHttp;
+
+        this.licenseId = this.$scope.currentActionData.license;
+    }
+
+    $onInit () {
+        this.loaders = {
+            licenseEnum: false,
+            userOrder: false
+        };
+
+        this.license = null;
+        this.number = 1;
+
+        this.$scope.orderUser = () => this.orderUser();
+
+        this.getLicenses(this.licenseId);
+    }
+
+    orderUser () {
+        this.loaders.userOrder = true;
+        this.licenseService.gotToOrderPrepaidLicenses(this.licenseId, this.license, this.number);
```

Comment:
The service changes the window or opens a new tab ? I don't know about that, what do you think @jleveugle ?